### PR TITLE
[WIP] Announce-Darga5

### PR DESCRIPTION
**Pull request by JPrause**
Announcement for Darga 5 release.

### CODE REVIEW

**Comment by JPrause** [comment]
@mfeifer here's the PR for darga5.

**Comment by mfeifer** [comment]
Here's some text for you start with in announcment.

I would mention:

- SmartState: Support analysis of VMs residing on NFS41 datastores
- User Interface:
    - Use VMRC desktop client if no NPAPI plugin is available
   - Add Advanced Search to Containers explorer

Fixed issues with Alerts
  - Alerts
    - Fixed alerts not sending SNMP v1 traps
    - Fixed users not being able to add alerts


Here is the full list of the things added and fixed in Darga 5

## Added
- Platform
  - Chargeback: Vm group by tag
  - API: Fix creation of Foreman provider
- Providers
  - SCVMM - Set CPU sockets and cores per socket
  - RHEVM: Report manufacturer and product info for RHEVM hosts
- SmartState: Support analysis of VMs residing on NFS41 datastores
- User Interface
  - Added rate assigning by tags to Container images
  - Allow bring VM out of retirement from detail page


## Fixed
- Automate
  - Alerts
    - Fixed alerts not sending SNMP v1 traps
    - Fixed users not being able to add alerts
  - Domain
    - Broken ordering of ae_domains for root tenant
    - Fixed problem with request_pending email method.
  - Provisioning
    - Set zone when deliver a service template provision task
    - Allow VMware provision without needing to specify a host
  - Service Model: Expose ems_events to Vm service model
- Platform
  - Security: Perform RBAC user filter check on requested ids before allowing request
- User Interface
  - Use VMRC desktop client if no NPAPI plugin is available
  - Add Advanced Search to Containers explorer
  - Save Authentication status on a Save

**Comment by JPrause** [comment]
Cool! Thanks @mfeifer I'll make those changes shortly.

File: site/_posts/2016-12-05-Announcing-Darga-5.html.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+title: Announcing Darga-5
+author: jprause
+date: 2016-12-05 08:35:39 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+We've just built Darga-5 This release contains a security fix, bug fixes, numerous UI tweaks, and stabilization.
+
```
Comment (mfeifer) [inline site/_posts/2016-12-05-Announcing-Darga-5.html.md]: need period after Darga-5

**Pull request merged**